packages/api-client/src/components/CommandPalette/CommandPaletteImport.vue

```diff
@@ -59,18 +63,28 @@ const { open: openSpecFileDialog } = useFileDialog({
   accept: '.json,.yaml,.yml',
 })
 
+watch(isInputUrl, (newVal) => {
+  if (!newVal) {
+    watchForChanges.value = false
+  }
+})
+
+watch(inputContent, (newVal) => {
+  if (!isUrl(newVal)) {
+    watchForChanges.value = false
+  }
+})
```

Comment:
Need comments on these. Also strongly prefer one liners for these type of statements:
 ```ts
if (!isUrl(newVal)) watchForChanges.value = false
```

Reply:
agreed

Reply:
but also that should be a lint rule

Reply:
+1 for one liner lint rule